src/LookupTable/hooks/useGetSingleTableStructure.hook.tsx

```diff
@@ -1,44 +0,0 @@
-import { ApolloQueryResult, QueryLazyOptions } from '@apollo/client'
-import { useEffect, useState } from 'react'
-import { useGetLookupTableStructureByIdLazyQuery } from '../../utils/generated/graphql'
-import { LookUpTableType } from '../types'
-
-type LookupTableStructureType = {
-  setStructureID: (id: number) => void
-  structureLoadState: ApolloQueryResult<any>
-  getStructure?: (options?: QueryLazyOptions<any>) => void
-  structure?: LookUpTableType
-}
-
-const useGetTableStructure = (): LookupTableStructureType => {
-  const [structureID, setStructureID] = useState<number>(0)
-  const [structure, setStructure] = useState<LookUpTableType>()
-
-  const [execute, structureLoadState]: [(options?: any) => void, ApolloQueryResult<any>] =
-    useGetLookupTableStructureByIdLazyQuery({
-      variables: { lookupTableID: structureID },
-      fetchPolicy: 'network-only',
-    })
```

Comment:
This "LazyQuery" added a whole lot of unnecessary complexity that seemed completely unjustified. The only reason I could see for it was to allow to re-loading the query after doing the "Import", but we can just do this with "refetch" in a much simpler (and more consistent with the rest of our app) way.